### PR TITLE
[CGPROD-2024] Remove onresize debounce

### PR DESCRIPTION
**Pull request by AdamXA**

### CODE REVIEW

File: build-scripts/licensechecker/licensewhitelist.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "Apache*",
     "Unlicense",
     "Zlib",
+    "CC0-1.0",
```
Comment (AnthonyGoodchild) [inline build-scripts/licensechecker/licensewhitelist.json]: confirmed ok?

File: src/core/scaler.js [hunk 2]
```diff
@@ -44,5 +44,5 @@ export function init(stageHeight, game) {
     );
 
     resize();
-    window.onresize = fp.debounce(200, resize);
+    window.onresize = resize;
```
Comment (AnthonyGoodchild) [inline src/core/scaler.js]: this rings a bell, are you sure it doesn't break anything?

Comment (AdamXA) [reply src/core/scaler.js]: will be tested to see if it breaks anything, don't know what it used to break, but using it breaks something else too!

Comment (AnthonyGoodchild) [reply src/core/scaler.js]: I've looked through the history but all the commits are vague :p

File: build-scripts/licensechecker/licensewhitelist.json (hunk 1 above)
Comment (AdamXA) [reply build-scripts/licensechecker/licensewhitelist.json]: I'm not sure who to confirm this with! The license seems fine?

**Comment by AnthonyGoodchild** [comment]
> Looks amazing! 🦍 🐒🐃 🐪🦏🐿 🦔 🦘🦆 🦅 🐋 🐌

There's more animals in your approval than characters in this PR

**Pull request merged**